### PR TITLE
GSOD 2024 page added

### PR DESCRIPTION
**Pull request by iArchitSharma**
**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
Waiting for build

File: src/collections/programs/gsod-2024/index.mdx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+### Previous experience with technical writers or documentation
+Our mentors have managed teams of technical writers working on documenting enterprise-grade software at large technology companies (Cisco, Seagate, SolarWinds). During the span of time, he has worked with technical writers in DITA and post-DITA environments (from Word to FrameMaker, structured writing, online help, various CMSes, git). Our mentors have worked with technical writers on documentation strategy, creating document sets, covering the full spectrum of reader personas.
+### Previous participation in Season of Docs, Google Summer of Code or others:
+Layer5 and its mentors have participated as mentors in MLH, FOSSASIA, GSoD 2020, GSoC 2019, 2020, 2021, 2022 and are in 2024 under CNCF. Mentoring and seeing others learn and grow is what Layer5 is all about - it’s a platform for mentees to hoist themselves up on and leapfrog from. We’re a platform for shared success.
```
Comment (Yashsharma1911) [inline src/collections/programs/gsod-2024/index.mdx:194]:
```suggestion
Layer5 and its mentors have participated as mentors in MLH, LFX, FOSSASIA, GSoD 2020, GSoC 2019, 2020, 2021, 2022 and are in 2024 under CNCF. Mentoring and seeing others learn and grow is what Layer5 is all about - it’s a platform for mentees to hoist themselves up on and leapfrog from. We’re a platform for shared success. 
```

Comment (Yashsharma1911) [reply src/collections/programs/gsod-2024/index.mdx:194]: I missed adding it

**Comment by l5io** [comment]
🚀 Preview for commit 7424f9e2e47cb490368dcccab7bb2c38da74c1da at: https://660b091c18967f0a05b1ef52--layer5.netlify.app

File: src/collections/programs/gsod-2024/index.mdx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+## How to Apply
+
+We interact daily over [Slack](https://slack.layer5.io/)
+
+Interested writers please contact us through #documenation Layer5 slack channel, or mail us at community@layer5.io
+
+If you have further question on participating or how to get started, please ask them in any public Layer5 slack channel so that others can also learn from answers to your questions, please visit Layer5 Discuss Forum for previous questions asked by other contributors.
+
+Visit official technical writer guide for more information
+## Project Ideas
```
Comment (Yashsharma1911) [inline src/collections/programs/gsod-2024/index.mdx:56]:
```suggestion
## Project Ideas

```

File: src/collections/programs/gsod-2024/index.mdx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+### Meshery’s users facing doc audit and update
```
Comment (Yashsharma1911) [inline src/collections/programs/gsod-2024/index.mdx:57]:
```suggestion
### Meshery’s users facing doc audit and update

```

**Comment by l5io** [comment]
🚀 Preview for commit 5f0a77bda49bb020c3db5f458f694fc40ab94396 at: https://660b0dfa16297213ca133a88--layer5.netlify.app

**Review by Yashsharma1911 — APPROVED** [review]
Thankyou @iArchitSharma

**Pull request merged**